Demo/Demo.swift

```diff
@@ -87,4 +87,18 @@ enum FinniversKitViews: String {
             .consentViewDemoView,
         ]
     }
+
+    static var lastSelectedView: FinniversKitViews? {
+        get {
+            guard let lastSelectedViewRawValue = UserDefaults.standard.value(forKey: "lastSelectedView") as? String else {
```

Comment:
Could you abstract "lastSelectedView" into a separate variable so we don't have it hardcoded twice?

Reply:
Absolutely 😊